plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -584,19 +636,77 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/user_status/api/v[0-9]+/hea
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
     
+# while trying to update the Weather location
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/weather_status/api/v[0-9]+/location" \
+    "id:9508802,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+# When trying to update user status message
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/user_status/api/v[0-9]+/user_status/message" \
+    "id:9508803,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
+
 
 #
 # [ Users ]
 #
 
-# Allow users to be deleted
+# Allow users to be deleted and user profile settings to be updated
 SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/cloud/users/" \
     "id:9508850,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
+
+# Allow users Profile visibility settings to be updated in Personal info
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/profile" \
+    "id:9508851,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+
+# [ Survey Client ]
+#
+
+# FP when trying to reject the survey client
```

Comment:
Not sure whether `cancel` is better than `reject` here, as I don't know the context.
```suggestion
# When trying to cancel the survey client
```

Reply:
The Survey client pops up in a notification asking whether you wan to accept or reject the survey which'll sends statistics about your Nextcloud Installation.

Reply:
I've updated the suggestion